arrays/c-or-cpp/unique-elements-in-an-array.c

```diff
@@ -0,0 +1,34 @@
+#include<stdio.h>
+
+int main(){
+    int a[1000];
```

Comment:
Prefer creating it a dynamic array and create it according to what size user enters.